config/storage/config.go

```diff
@@ -32,6 +38,12 @@ func Configure(p *config.Provider) {
 			},
 		}
 		r.UseAsync = true
```

Comment:
Do these really need to be async? I know they have long timeouts in Terraform but almost 95% of resources in native providers are sync in the sense that once creation is completed, the resource is ready. Not a big deal though, we can always change it.

Reply:
I have not timed them. Not sure how long it takes to provision them. Maybe as we populate examples, we can have rough estimates and switch to synchronous mode for them. Async looks like a safer choice for me.